shared/macros.jinja

```diff
@@ -1745,3 +1745,44 @@ Fill in <tt>GRUBENV_FILE_LOCATION</tt> based on information above.
 
     $ systemctl enable --now {{{ service }}}
 {{%- endmacro -%}}
+
+{{#
+    Describe how to check a kernel compile parameter
+
+:param config: The kernel config parameter
+:type config: str
+:param value: The value for the given config
+:type value: str
+
+#}}
+{{% macro kernel_build_config_describe_config(config, value) -%}}
+    The configuration that was used to build kernel is available at <tt>/boot/config-*</tt>.
+    To check the configuration value for <tt>{{{ config }}}</tt>, run the following command:
+    <tt>grep {{{ config }}} /boot/config-*</tt>
+{{%- endmacro %}}
```

Comment:
I still think we should update these two macros to handle the case where value is set to `n`. For example,

```suggestion
{{% macro kernel_build_config_describe_config(config, value) -%}}
    The configuration that was used to build kernel is available at <tt>/boot/config-*</tt>.
    To check the configuration value for <tt>{{{ config }}}</tt>, run the following command:
    <tt>grep {{{ config }}} /boot/config-*</tt>
    
    {{% if value == 'n' %}}
    The result should return empty or a commented line
    {{% endif %}}}
{{%- endmacro %}}
```

Reply:
Forgot about that

Reply:
Updated both macros in [10017fb](https://github.com/ComplianceAsCode/content/pull/8435/commits/10017fb29dae9bd3958ac786a426d9f83682f2b0)